### PR TITLE
Create nico.json

### PR DESCRIPTION
**Pull request by NicoGaming77**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [ ] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/nico.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "repo": "https://github.com/NicoGaming77/NicoGaming77.github.io",
+  "owner": {
+    "username": "NicoGaming77",
+    "email": "nicodreyer7@icloud.com",
+    "discord": "nicosnico#7674"
+  }
+}
+
+{
+  "record": {
```
Comment (mtgsquad) [inline domains/nico.json:12]: Invalid JSON, the record object should come after owner, not in a seperate thing (forgot what it's called)

**Review by mtgsquad — DISMISSED** [review]

File: domains/nico.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal Website",
+  "repo": "https://github.com/NicoGaming77/NicoGaming77.github.io",
+  "owner": {
+    "username": "NicoGaming77",
+    "email": "nicodreyer7@icloud.com",
+    "discord": "nicosnico#7674"
+  }
```
Comment (MouseMoosz) [inline domains/nico.json:8]: There should be a comma after each object unless there aren't any more objects.

**Review by VaibhavSys — CHANGES_REQUESTED** [review]

File: domains/nico.json (hunk 2 above)
Comment (MouseMoosz) [reply domains/nico.json:8]: JSON doesn't support trailing commas, so remove the comma after `"discord"` key, I was asking you to put a comma after the owner object key.

**Review by MouseMoosz — CHANGES_REQUESTED** [review]

File: domains/nico.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal Website",
+  "repo": "https://github.com/NicoGaming77/NicoGaming77.github.io",
+  "owner": {
+    "username": "NicoGaming77",
+    "email": "nicodreyer7@icloud.com",
+    "discord": "nicosnico#7674",
```
Comment (MouseMoosz) [inline domains/nico.json:7]: JSON doesn't support trailing commas, so remove the comma after `"discord"` key, I was asking you to put a comma after the owner object key.

**Review by VaibhavSys — CHANGES_REQUESTED** [review]

File: domains/nico.json (hunk 3 above)
Comment (NicoGaming77) [reply domains/nico.json:7]: I'm sorry but I don't really understand what you mean by "after the owner object key", I'm not very familiar with JSON.

Comment (NicoGaming77) [reply domains/nico.json]: Wait I think I got it

**Comment by omxpro** [comment]
Yep you resolved it

**Review by VaibhavSys — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**